### PR TITLE
Rintu adminSubCategory add and edit

### PR DESCRIPTION
**Pull request by rinturejikc**

### CODE REVIEW

**Review by JaseelaAlimahamed — APPROVED** [review]

File: src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          <h2 className="text-2xl font-bold   text-dark_blue mt-3 ml-2">Type</h2>
+          <svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M11 4.00023H4C3.46957 4.00023 2.96086 4.21094 2.58579 4.58601C2.21071 4.96109 2 5.46979 2 6.00023V20.0002C2 20.5307 2.21071 21.0394 2.58579 21.4144C2.96086 21.7895 3.46957 22.0002 4 22.0002H18C18.5304 22.0002 19.0391 21.7895 19.4142 21.4144C19.7893 21.0394 20 20.5307 20 20.0002V13.0002M18.5 2.50023C18.8978 2.1024 19.4374 1.87891 20 1.87891C20.5626 1.87891 21.1022 2.1024 21.5 2.50023C21.8978 2.89805 22.1213 3.43762 22.1213 4.00023C22.1213 4.56284 21.8978 5.1024 21.5 5.50023L12 15.0002L8 16.0002L9 12.0002L18.5 2.50023Z"
```
Comment (JaseelaAlimahamed) [inline src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx:42]: make a file in public or asset foe this svg  and call it here

File: src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+              stroke="#1E1E1E"
```
Comment (JaseelaAlimahamed) [inline src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx:43]: usetailwind colors

File: tailwind.config.js [hunk 3]
```diff
@@ -20,7 +20,8 @@ module.exports = {
       colors: {
         primary: "#FFFFFF",
         secondary: "#000000",
-
+        gray:"#E5E7EB",
+        id_gray:"#797C7B",
```
Comment (JaseelaAlimahamed) [inline tailwind.config.js:24]: check this colors already mention in develop baranch then use it

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 3 above)
Comment (rinturejikc) [reply tailwind.config.js:24]: ok

File: src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx (hunk 2 above)
Comment (rinturejikc) [reply src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx:43]: ok

File: src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx (hunk 1 above)
Comment (rinturejikc) [reply src/components/franchaseeComponets/Edit/Add Franchasee Type.jsx:42]: ok

**Comment by rinturejikc** [comment]
resolved

File: src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+          className="bg-violet text-primary px-3 py-1 rounded-full"
+        >
+          Save
+        </button>
+      </div>
+    </div>
+  );
+};
+
+export default NewSubCategory;
```
Comment (JaseelaAlimahamed) [inline src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx:185]: dont write this much code in one component

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]

File: src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+          onClick={handleSave}
+          className="bg-violet text-primary px-3 py-1 rounded-full"
+        >
+          Save
+        </button>
+      </div>
+    </div>
+  );
+};
+
```
Comment (JaseelaAlimahamed) [inline src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx:184]: make it differnt component

**Pull request merged**